pkg/mixin/pkgmgmt_test.go

```diff
@@ -16,12 +16,12 @@ func TestRunner_BuildCommand(t *testing.T) {
 		runnerCommand string
 		wantCommand   string
 	}{
-		{"build", "build", "/home/myuser/.porter/mixins/exec/exec build"},
-		{"install", "install", "/home/myuser/.porter/mixins/exec/exec install"},
-		{"upgrade", "upgrade", "/home/myuser/.porter/mixins/exec/exec upgrade"},
-		{"uninstall", "uninstall", "/home/myuser/.porter/mixins/exec/exec uninstall"},
-		{"invoke", "status", "/home/myuser/.porter/mixins/exec/exec invoke --action status"},
-		{"version", "version --output json", "/home/myuser/.porter/mixins/exec/exec version --output json"},
+		{"build", "build", "/home/myuser/.porter/mixins/exec/exec build\n\\home\\myuser\\.porter\\mixins\\exec\\exec.exe build"},
```

Comment:
what is happening here?

Reply:
You can separate expected output by a newline character, so for this test there are two possible expected outputs which both pass

Reply:
oh interesting, might be easier to read if it is separated into array for the want command? maybe it isn't possible, not sure how the testing framework works.